README.md

```diff
@@ -98,7 +98,7 @@ However, there are some instances where you wish to override this, e.g.
 To help with this, there are two mechanisms for overriding the tinymce.min.js URL.
 
  1. Set `tinymce_live_demo_url` in `_config-local-override.yml`.
```

Comment:
This doesn't work anymore... if you want to override it, you need to add something like this to the antora dev playbook instead:
```
  attributes:
    tinymce_live_demo_url: https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/6-dev/tinymce.min.js
```